ASSIGNMENT_1 SUBMISSION

Malekikh opened the pull request:
Uploaded modified md file and python to include the few updates to the python code

## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
assignment 1 submission
## What did you learn from the changes you have made?
sampling types for a given example
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
-
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
-
## How were these changes tested?
VS code
## A reference to a related issue in your repository (if applicable)
-
## Checklist
- [x] I can confirm that my changes are working as intended

Malekikh committed:
ASSIGNMENT_1 SUBMISSION
Uploaded modified md file and python to include the few updates to the python code

PatelVishakh reviewed (changes requested):
Assignment 1 Incomplete

Comments:

Portion 1: Should be explaining the sampling procedure, sample size, sampling frame, any underlying distributions involved for each line in the code.

Portion 2: Samping methodology is the same
Implement the random seed function BEFORE doing 100 repetitions so those ones are reproducible

Malekikh committed:
MODIFIED ASSIGNMENT AS REQUESTED

I made changes to portion 1 and 2 as instructed.
For portion 1:
provided more clarification on the sampling procedure and showing it on the python code line by line.
For portion 2:
looked at R code of the blog and noticed that both approaches are using same sampling methodologies and producing same results.

PatelVishakh reviewed (approved):
Assignment 1 complete
